src/main/java/com/revature/studyforce/flashcard/repository/FlashcardRepository.java

```diff
@@ -1,21 +1,21 @@
 package com.revature.studyforce.flashcard.repository;
 
-import com.revature.StudyForce.flashcard.model.Flashcard;
+import com.revature.studyforce.flashcard.model.Flashcard;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
-import java.util.List;
-
 /**
- * Spring repository for Flashcards
- *
+ * Repository for Flashcard model
  * @author Luke
  */
 @Repository
 public interface FlashcardRepository extends JpaRepository<Flashcard, Integer> {
 
-    // Find questions by difficulty
```

Comment:
Interfaces should not be commented like this, these are generated by spring so you have no knowledge of their inner workings and cannot provide accurate documentation.